src/references/rainbow-token-list/index.ts

```diff
@@ -84,13 +97,9 @@ async function readRNFSJsonData<T>(filename: string): Promise<T | null> {
   }
 }
 
-async function writeRNFSJsonData<T>(filename: string, data: T) {
+async function writeMMKVJsonData<T>(filename: string, data: T) {
   try {
-    await RNFS.writeFile(
-      path.join(RNFS.CachesDirectoryPath, filename),
-      JSON.stringify(data),
-      'utf8'
-    );
+    await Promise.resolve(storage.set(filename, JSON.stringify(data)));
```

Comment:
no need for async, no need for promise

Reply:
I didn't want to rewrite it completely because we have plans to get rid of it for good anyway. So i made the smallest amount of changes to use mmkv as possible.

Reply:
Hey, I think this refactor is not big so I prefer not to compromise the code quality. We don't use this in many places so please make it properly.

We have plans to get rid of this, but the timeline is not clear

Reply:
I agree that the way you made those changes minimize the changelog, but IMO there's little profit from keeping those one time changes small, and by introducing artificial async this code becomes much much harder to read and comprehend.

I would much prefer to rewrite it in a simplest way possible or otherwise someone in future will have very hard time figuring out what were the secret performance reasons for doing it this mysterious way.